website/docusaurus.config.js

```diff
@@ -0,0 +1,141 @@
+ * - If it matches one of the hardcoded prefixes make it a GitHub link.
+ */
+function remarkPluginFixLinksToRepositoryArtifacts() {
+  /** @type {import('unified').Transformer} */
+  const transformer = async (ast, file) => {
+    // Package does not support require es modules.
+    // Easiest workaround I found.
+    let { visit } = await import('unist-util-visit');
+    visit(ast, 'link', (node) => {
+      /** @type string */
+      let link = node.url;
+      if (link.startsWith('http://') || link.startsWith('https://')) {
+        return;
+      }
+      let repoRoot = path.resolve(__dirname, '..');
+      let absolute = path.resolve(repoRoot, file.dirname, link);
+      let relative = path.relative(__dirname, absolute);
+      if (relative.startsWith('src')
+        || relative.startsWith('cursorless-snippets')
+        || relative.startsWith('.github')) {
+        const repoLink = 'https://github.com/cursorless-dev/cursorless-vscode/tree/main/';
+        const linkToRepositoryArtifact = repoLink.concat(relative);
+        node.url = linkToRepositoryArtifact;
+      }
+    });
+  };
+  return transformer;
+};
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'Cursorless',
+  tagline: 'Never use the cursor again',
+  // GitHub pages url
+  url: 'https://cursorless-dev.github.io/',
+  baseUrl: '/cursorless-vscode/',
+  onBrokenLinks: 'throw',
+  onBrokenMarkdownLinks: 'warn',
+  favicon: 'img/favicon.ico',
+  organizationName: 'pokey',
+  projectName: 'cursorless',
+
+  staticDirectories: [
+    'static',
+  ],
+
+  plugins: [
+    [
+      'docusaurus-plugin-typedoc',
+
+      // Plugin / TypeDoc options
+      {
+        entryPoints: ['../src/'],
+        entryPointStrategy: 'expand',
+        readme: 'none',
+        tsconfig: '../tsconfig.json',
+        out: 'contributing/api',
+        plugin: [
+          'typedoc-plugin-rename-defaults', 
+          'typedoc-plugin-mdn-links',
+          // typedoc generates using <internal> tag that is not closed. 
+          // MDX in docusaurus does not like that and reports unclosed jsx tag.
+          // 'typedoc-plugin-missing-exports'
+        ]
+      }
+    ],
+  ],
+
+  presets: [
+    [
+      'classic',
+      /** @type {import('@docusaurus/preset-classic').Options} */
+      ({
+        docs: {
+          path: 'docs',
+          routeBasePath: '/',
+          sidebarPath: require.resolve('./sidebar.js'),
+          beforeDefaultRemarkPlugins: [remarkPluginFixLinksToRepositoryArtifacts],
+        },
+      })
+    ],
+  ],
+
+  themeConfig:
+    /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
+    ({
+      navbar: {
+        title: 'Cursorless',
+        logo: {
+          alt: 'Cursorless',
+          src: 'icon.svg',
+        },
+        items: [
+          {
+            position: 'left',
+            type: 'doc',
+            docId: 'user/README',
+            to: '/',
```

Comment:
how come this one is to `/` and the other is to `contributing/`?  should this not link to `user/`?